packages/components/src/dropdown-menu/index.js

```diff
@@ -12,34 +12,42 @@ import { DOWN } from '@wordpress/keycodes';
 /**
  * Internal dependencies
  */
+import DropdownMenuSeparator from './separator';
 import IconButton from '../icon-button';
 import Dropdown from '../dropdown';
 import { NavigableMenu } from '../navigable-container';
 
 function DropdownMenu( {
+	contentClassName,
+	children,
+	className,
+	controls,
 	icon = 'menu',
 	label,
+	labelPosition,
```

Comment:
I think I added `contentClassName` and `labelPosition` as well. This needs to be reviewed as I brought them for backward compatibility.

Reply:
The current list of new props which might need to be documented:
- `labelPosition`
- `menuClassName`
- `popoverClassName`
- `toggleClassName`

All 3 class names props exist only for backward compatibility. I'm hesitant whether we should document it all.

Reply:
Interesting, it hadn't occurred to me the need to provide customization options for the rendered toggle. My next thought was whether we could just expose `renderToggle` as a render prop to be customized by the consuming renderer. I guess the problem with that is how to manage on their behalf some of the best practice behaviors (i.e. ArrowDown to open), as I don't think it would be wise to leave open the option that this be possible to overlook (footgun).

Reply:
Understanding the desire to retain existing behaviors, I do wonder how we could limit or consolidate the customization options. Something like `labelPosition`, for example, seems to only exist where otherwise if we allow the standard "top" behavior to apply for the "More Menu", it might escape the page in a way which is undesirable. But it's the sort of thing we try to build-in to the behavior of Popover (auto-reverse when otherwise the popover would exceed bounds), so it seems like something which should be programmatically determined.

Reply:
How about we prefix all of them with `__untable` to conclude this PR and we attack it later with some nice automation as discussed above. I don't feel good about blocking this PR as it's solving a couple of issues at once and gives a better chance to tackle further a11y issues.

Reply:
> How about we prefix all of them with `__untable` to conclude this PR and we attack it later with some nice automation as discussed above. I don't feel good about blocking this PR as it's solving a couple of issues at once and gives a better chance to tackle further a11y issues.

I would be okay with this 👍

Reply:
Applied in c25dc8a.